### ISSUE TITLE
LDAP container deployment Failing

### ISSUE THREAD

**Issue by shouro** [GluuFederation/gluu-flask#58]
During XXX training LDAP nodes are failed to deploy.
They were using AWS vms with 2 cores and 4GB ram which is lower than our hardware requirements.
http://www.gluu.org/docs-cluster/admin-guide/overview/#hardware-requirements

We also got this errors rarely in our testing but not always.
(In my testing in XXXX which is also in AWS, This errors resolved with a simple vm restart.)
In our testing, we can't always reproduce this errors. It is still unclear when this error
will appear. But in my testing, I get this error only in this 2 cases,
[1] ldap import command
[2] ldap index command
sometimes import passed and index failed or sometimes import failed.

But bottom line is, we must do something about it, priority is very high.

**Comment by iromli** [comment]
Right, let me do tests on VM with lower resource

**Comment by nynymike** [comment]
It is silly to not look at this ldif file. The more likely problem is that there is a problem with the ldif file. Also, please do not reference customers on github.

**Comment by iromli** [comment]
In certain cases, the ldif file is not copied to the node, hence at the
time import-ldif executed, this command complains about missing file.

But I will double check the generated ldif files.

**Comment by iromli** [comment]
After several tests, I figured out that sometimes files are not copied into the node. This makes indexing/import ldif failed. In latest codebase, deployment script checks whether those files are already copied into the node or not. When they are missing in node FS, this script will try to re-copy the files (max. retry attempt is 5 times within 30 secs). 

I have tested the newest beta release in digitalocean VM (4 GB RAM and 2 CPUs). The result is, although it's slower than previous version (due to retry attempt), I no longer see the I/O problem when doing indexing/import ldif process.

Since I don't have access to AWS, I would appreciate if someone test the beta release in AWS or provide me access to the VM so I can test the release by myself.
